frontend/src/pages/_app.tsx

```diff
@@ -58,17 +58,26 @@ const REACT_TOUR_OPTIONS: Omit<MapTourProviderProps, 'children'> = {
 export type ResilienceAppProps = {
   dehydratedState?: DehydratedState;
   translations: Translations;
+  published: boolean;
   setTranslations?: (translations: Translations) => {
     type: string;
     translations: Translations;
   };
   dispatch?: (action: unknown) => void;
 };
 
+export type JourneyPageProps = ResilienceAppProps & {
+  published: boolean;
+};
```

Comment:
If there's a `published` attribute in `ResilienceAppProps`, why is it necessary to redefine it here?

I think it would be best to keep the attribute in `JourneyPageProps` alone.

Reply:
Ah, I forgot to remove it there. Thanks!